kani-compiler/src/codegen_boogie/overrides/hooks.rs

```diff
@@ -0,0 +1,124 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This module contains various codegen hooks for functions that require special handling.
+//!
+//! E.g.: Functions in the Kani library that generate assumptions or symbolic variables.
+//!
+//! It would be too nasty if we spread around these sort of undocumented hooks in place, so
```

Comment:
This comment is a bit odd. I know we have that in the `goto`, but maybe it doesn't need to be replicated.

Reply:
Removed.